app/Filament/Resources/SearchRecipeResource/Pages/CreateSearchRecipe.php

```
@@ -1,12 +1,27 @@
 
 class CreateSearchRecipe extends CreateRecord
 {
     protected static string $resource = SearchRecipeResource::class;
+
+    /**
+     * This method is executed after a record has been successfully created.
+     */
+    protected function afterCreate(): void
+    {
+        $searchRecipe = $this->record;
+        LaunchSearchRecipeJob::dispatch($searchRecipe);
```

PHPStan (2.0.3)

Job class App\Jobs\LaunchSearchRecipeJob constructor invoked with 1 parameter in App\Jobs\LaunchSearchRecipeJob::dispatch(), 0 required.


(arguments.count)